docs/sdks/sdk-ref-python.md

```diff
@@ -209,6 +209,8 @@ If you're using server-side integration (see [Publisher Integration Guide, Serve
    If the user has opted out, this method returns `None`, indicating that the user's identity should be removed from the session. To confirm optout, you can use the `token_refresh_response.is_optout()` function.
 
 ## Usage for Advertisers/Data Providers
+### Map DII to raw UID2s
+To mapp email addresses, phone numbers, or their respective hashes to their raw UID2s and salt bucket IDs.
```

Comment:
mapp update to map
Suggest:
To map email addresses, phone numbers, or their respective hashes to their raw UID2s and salt bucket IDs, follow these steps: